src/dev-mock-service/README.md

```diff
@@ -0,0 +1,47 @@
+## Dev Mock Service
+
+Dev Mock Service Worker is an API mocking library that allows you to specify 
+custom responses for any API calls inside your app.
+
+### Capabilities
+
+- Mock API requests
+- Dev/Prod switch
+
+### Installation
+
+Add the line `import { DevMockMode, DevMock } from '@devvit/kit';` in the beginning of your root component.
+
+### API mocks
+
+With custom handler functions, you can override the response for any API call, such as
+Redis, RedditAPI, or HTTP request.
+
+- In Dev mode (`DevMockMode.Dev`), handlers are applied for all requests with the matching method and ID (if available).
+- In Prod mode (`DevMockMode.Prod`), handlers are ignored.
```

Comment:
this will encourage users to ship development-only code which may be large. I'd love to see this code fall out of apps for at least non-prerelease builds so there wasn't a wrong way to use it.

Reply:
How can I achieve this?

Reply:
I'd like the introduction of a `IS_PROD` or `NODE_ENV` or `IS_PLAYTEST` environmental variable that could be statically added at build time for tree shaking. Would be useful for more than just here